requirements.txt

```diff
@@ -4,4 +4,5 @@ pillow
 pyramid
 pyramid_jinja2
 typing_extensions
-werkzeug
\ No newline at end of file
+werkzeug
+waitress
```

Comment:
Please sort 🔡.

Reply:
BTW, can black do automatic sorting?

Reply:
Don't think so. But vscode can do it manually for a selection.

Reply:
I will solve it in the other PR.

Reply:
> Don't think so. But vscode can do it manually for a selection.

Nice, I get it now: https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines

Reply:
I think there's native support too in vscode.

Reply:
Oh, really, how do you use/enable that?

Reply:
I didn't activate anything that I know of. Just select a range of lines to sort, then activate the command palette and type `sort lines ascending`. You'll see the command.

Reply:
I had to install that extension, otherwise I don't see such option.